src/__data__/languages.json

```diff
@@ -73,5 +73,19 @@
         "url": "https://github.com/signalhunter"
       }
     ]
+  },
+  "zh_CN": {
+    "name": "Chinese (Simplified)",
+    "nativeName": "简体中文",
+    "country": "cn",
+    "antLocale": "zh_CN",
+    "dayjsLocale": "zh",
+    "timeagoLocale": "zh",
```

Comment:
Unless I'm mistaken, I believe the timeago locale is `zh-CN`: 
https://github.com/nmn/react-timeago/blob/master/src/language-strings/zh-CN.js

Day.js also has `zh` and `zh-cn`, which have some tiny differences (`relativeTime.future`), not sure which one is correct here, I'll leave that to your judgement:
https://github.com/iamkun/dayjs/blob/dev/src/locale/zh-cn.js
https://github.com/iamkun/dayjs/blob/dev/src/locale/zh.js

Reply:
Ahh ok I didn't know where those locales really came from, but yeah I'll change that

Reply:
For future reference to anyone reading this, they're documented in the translating section in the README: https://github.com/tmpim/KristWeb2#contributing-translations